### PR TITLE
Merge stable 5.0

### PR DESCRIPTION
**Pull request by dragonchaser**
This PR attempts to merge the changes in master back into `stable-5.0`

### CODE REVIEW

File: tests/acceptance/expected-failures-localAPI-on-OCIS-storage.md [hunk 1]
```diff
@@ -92,16 +92,16 @@ The expected failures in this file are from features in the owncloud/ocis repo.
...
-- [apiGraph/addUserToGroup.feature:154](https://github.com/owncloud/ocis/blob/master/tests/acceptance/features/apiGraph/addUserToGroup.feature#L154)
-- [apiGraph/addUserToGroup.feature:188](https://github.com/owncloud/ocis/blob/master/tests/acceptance/features/apiGraph/addUserToGroup.feature#L188)
-- [apiGraph/addUserToGroup.feature:189](https://github.com/owncloud/ocis/blob/master/tests/acceptance/features/apiGraph/addUserToGroup.feature#L189)
-- [apiGraph/addUserToGroup.feature:190](https://github.com/owncloud/ocis/blob/master/tests/acceptance/features/apiGraph/addUserToGroup.feature#L190)
+- [apiGraph/addUserToGroup.feature:184](https://github.com/owncloud/ocis/blob/master/tests/acceptance/features/apiGraph/addUserToGroup.feature#L184)
+- [apiGraph/addUserToGroup.feature:185](https://github.com/owncloud/ocis/blob/master/tests/acceptance/features/apiGraph/addUserToGroup.feature#L185)
+- [apiGraph/addUserToGroup.feature:186](https://github.com/owncloud/ocis/blob/master/tests/acceptance/features/apiGraph/addUserToGroup.feature#L186)
 - [apiGraph/createGroup.feature:42](https://github.com/owncloud/ocis/blob/master/tests/acceptance/features/apiGraph/createGroup.feature#L42)
```
Comment (saw-jan) [inline tests/acceptance/expected-failures-localAPI-on-OCIS-storage.md:105]: These need to be reverted, CI is complaining

File: tests/acceptance/expected-failures-localAPI-on-OCIS-storage.md [hunk 2]
```diff
@@ -114,24 +114,24 @@ The expected failures in this file are from features in the owncloud/ocis repo.
...
 
-- [apiGraph/addUserToGroup.feature:373](https://github.com/owncloud/ocis/blob/master/tests/acceptance/features/apiGraph/addUserToGroup.feature#L373)
-- [apiGraph/addUserToGroup.feature:387](https://github.com/owncloud/ocis/blob/master/tests/acceptance/features/apiGraph/addUserToGroup.feature#L387)
+- [apiGraph/addUserToGroup.feature:369](https://github.com/owncloud/ocis/blob/master/tests/acceptance/features/apiGraph/addUserToGroup.feature#L369)
+- [apiGraph/addUserToGroup.feature:383](https://github.com/owncloud/ocis/blob/master/tests/acceptance/features/apiGraph/addUserToGroup.feature#L383)
 
 ### [Adding the same user as multiple members in a single request results in listing the same user twice in the group](https://github.com/owncloud/ocis/issues/5855)
 
-- [apiGraph/addUserToGroup.feature:424](https://github.com/owncloud/ocis/blob/master/tests/acceptance/features/apiGraph/addUserToGroup.feature#L424)
+- [apiGraph/addUserToGroup.feature:420](https://github.com/owncloud/ocis/blob/master/tests/acceptance/features/apiGraph/addUserToGroup.feature#L420)
```
Comment (saw-jan) [inline tests/acceptance/expected-failures-localAPI-on-OCIS-storage.md:134]: same here as well
>These need to be reverted, CI is complaining

File: tests/acceptance/expected-failures-localAPI-on-OCIS-storage.md [hunk 3]
```diff
@@ -92,8 +92,6 @@ The expected failures in this file are from features in the owncloud/ocis repo.
 
 #### [Same users can be added in a group multiple time](https://github.com/owncloud/ocis/issues/5702)
 
-- [apiGraph/addUserToGroup.feature:289](https://github.com/owncloud/ocis/blob/master/tests/acceptance/features/apiGraph/addUserToGroup.feature#L289)
-
```
Comment (saw-jan) [inline tests/acceptance/expected-failures-localAPI-on-OCIS-storage.md:96]: this test can still fail. so we should have it here in expected-failure

**Comment by saw-jan** [comment]
@dragonchaser Sorry for the hassle, PR https://github.com/owncloud/ocis/pull/8114 was merged accidentally so the shareingNg pipeline is failing and I reverted it in https://github.com/owncloud/ocis/pull/8209

**Comment by saw-jan** [comment]
> @dragonchaser Sorry for the hassle, PR #8114 was merged accidentally so the shareingNg pipeline is failing and I reverted it in #8209

Revert PR is merged

**Review by micbar — APPROVED** [review]
Looks good!

**Review by phil-davis — COMMENTED** [review]
CI passes, and there are lots of acceptance test changes (so they must be passing except for what is in expected-failures files). That is good.

There are various changelog changes, and items added to the unreleased changelog. I suppose that is expected, and somebody will (one day) do a version 5 minor or patch release from the stable-5.0 branch which will "process" the unreleased changelog items.

The branch is called `stable-5.0` so I guess that means that any changes put into that branch must be fixes (patches), which will only result in a 5.0.1 5.0.2 etc. release. There are 2 changelog items that are called "enhancement", so someone should check that those are OK to be released in a patch-level release.

**Comment by dragonchaser** [comment]
@phil-davis the naming is imho confusing since we do not have a release 5.0 (yet)

**Review by phil-davis — APPROVED** [review]

**Pull request merged**